### PR TITLE
Add subpage files field query example

### PR DESCRIPTION
**Pull request by kevinvennitti**
Open to any edits or contributions :)

### CODE REVIEW

File: content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt [hunk 1]
```diff
@@ -104,6 +104,25 @@ The default (preview) image is the first image in the folder. You can configure
...
+
+```yaml
+image: page.my_files_field.toFile
+```
+
+<info>
+Note that if you have `multiple: true`, be sure to query only one image:
+
+```yaml
+image: page.my_multiple_files_field.first.toFile
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt]: This results in the same as above. `toFile` only ever returns a single file

File: content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt [hunk 2]
```diff
@@ -104,6 +104,25 @@ The default (preview) image is the first image in the folder. You can configure
 image: page.image.findBy("name", "cover")
 ```
 
+### Preview image from page files field
+
+You can use the image of each subpage `files` field as preview image:
+
+```yaml
+image: page.my_files_field.toFile
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt]: We don't use snake_case anywhere across the documentation. Could you please change to camelCase.

Comment (kevinvennitti) [reply content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt]: Done ✅

File: content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt (hunk 1 above)
Comment (texnixe) [reply content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt]: Your syntax is in any case not correct, would have to be

```
image: page.my_multiple_files_field.toFiles.first
```

File: content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt (hunk 2 above)
Comment (texnixe) [reply content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt]: Thanks!

**Pull request merged**